test/deprecations_test.rb

```diff
@@ -3,10 +3,12 @@
 require 'test_helper'
 
```

Comment:
Can we just `return if StatsD::Instrument.strict_mode_enabled?` here to avoid multiple checks later on?

Reply:
That would work, but I kind of like it explicit like this. The test suite runs in 0.1 seconds on my machine, so over-optimizing this is not really needed.